_posts/2023-11-30-indico-3-3-news.md

```diff
@@ -0,0 +1,94 @@
+---
+layout:     post
+title:      "Indico 3.3"
+date:       2023-11-30 12:00
+author:     "Indico Team"
+categories: indico release
+---
+
+We are happy to announce the release of Indico 3.3. This release brings a lot of new features and improvements, including a **new Check-in app**, **accessibility improvements (a11y)**, **receipt generation** and a whole lot more.
```

Comment:
> Today, we are introducing Indico 3.2! This upcoming release ...

from [here](https://getindico.io/indico/release/2022/06/10/indico-3-2-news.html) sounds nice as well IMHO

Reply:
if you want to go for the "at CERN" I'd mention "upcoming deployment". But I'd avoid it because the blog post is also targetting a more general audience.

Or maybe use something like "the version will be deployed at the CERN Indico instance soon, and released to the wider community at a later point" (not sure if really necessary though)

Reply:
I used the one from 3.2 - I think it sounds better than also mentioning CERN..